src/lib/swapper/LifiSwapper/getTradeQuote/getTradeQuote.ts

```diff
@@ -141,10 +134,11 @@ export async function getTradeQuote(
 
   const selectedRoute = lifiRoutesResponse.routes[SELECTED_ROUTE_INDEX]
 
-  const minimumCryptoHuman = toHuman({
-    value: getMinimumAmountFromRoutes(lifiRoutesResponse.routes, lifiBridges) ?? Infinity,
-    inputPrecision: sellLifiToken.decimals,
-  }).toString()
+  const minimumUsdHuman = getMinimumUsdHumanFromRoutes(lifiRoutesResponse.routes, lifiBridges)
```

Comment:
Can we make all the `human` across this commit precision amounts instead? The conversion to human should really only happen at the view-layer step.

Reply:
Can do that, however all of the minimum amounts from lifi need to be converted to usd using a "usd human" rate attached to the lifi tokens themselves. Unfortunately there doesnt seem to be a more precise method to convert the minimum amounts into the base currency without resorting to low-precision rates denominated in usd

Reply:
@woodenfurniture and I discussed a path forward on this to start from base units and convert when showing to the user, which will come in a follow-up PR, and slightly improve precision.